### PR TITLE
Removed broken and outdated link

### LINKED ISSUE #4246: Broken link in compiler-element page
# Broken link in compiler-element page

There is a broken link at the bottom of the compiler-element page.

## Issues with Existing Topics

https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/compiler/compiler-element

Suggest reviewers

    - If you know someone who can provide feedback, use '@' to ask them to review.
@jralexander

### PR DESCRIPTION
**Pull request by JRAlexander**
# Removed broken and outdated MSDN link

## Summary

Removed broken and outdated MSDN link at bottom of article

Fixes #4246

## Suggested Reviewers

If you know who should review this, use '@' to request a review.

### CODE REVIEW

File: docs/framework/configure-apps/file-schema/compiler/compiler-element.md [hunk 1]
```diff
@@ -108,5 +108,4 @@ Specifies the compiler configuration attributes for a language provider.
  <xref:System.CodeDom.Compiler.CodeDomProvider>  
  [Configuration File Schema](../../../../../docs/framework/configure-apps/file-schema/index.md)  
  [\<compilers> Element](../../../../../docs/framework/configure-apps/file-schema/compiler/compilers-element.md)  
- [Specifying Fully Qualified Type Names](../../../../../docs/framework/reflection-and-codedom/specifying-fully-qualified-type-names.md)  
- [compiler Element for compilers for compilation (ASP.NET Settings Schema)](http://msdn.microsoft.com/library/f7d6b078-5d42-4134-b3f7-62e1aba1df1e)
```
Comment (mairaw) [inline docs/framework/configure-apps/file-schema/compiler/compiler-element.md]: do we want to completely remove this or change to a version that works?
https://msdn.microsoft.com/en-us/library/f7d6b078-5d42-4134-b3f7-62e1aba1df1e(v=vs.100)

Comment (JRAlexander) [reply docs/framework/configure-apps/file-schema/compiler/compiler-element.md]: Lets err on the side of maintainability and add the version that works.

**Review by mairaw — APPROVED** [review]
LGTM pending build

**Comment by mairaw** [comment]
I added the deleted space so we have a break line between the links. Otherwise it looks like this:
image

**Pull request merged**